.github/workflows/ci.yml

```diff
@@ -319,7 +478,7 @@ jobs:
           export MAVEN_OPTS="${MAVEN_INSTALL_OPTS}"
           ./mvnw install ${MAVEN_FAST_INSTALL} -am -pl $(echo '${{ matrix.modules }}' | cut -d' ' -f1)
       - name: Maven Tests
-        run: ./mvnw test ${MAVEN_TEST} -pl ${{ matrix.modules }}
+        run: $RETRY ./mvnw test ${MAVEN_TEST} -pl ${{ matrix.modules }}
```

Comment:
I don't think we should retry the tests? That is just hiding flaky tests.

Reply:
removed it.